composer.json

```
@@ -1,6 +1,6 @@
 {
-    "name": "ozee31/cakephp-cors",
-    "description": "A CakePHP (3.3.x) plugin for activate cors domain in your application",
+    "name": "rabp99/cakephp-cors",
+    "description": "It's a pull request from ozee31/cakephp-cors that solve the issue about OPTIONS method. A CakePHP 4 plugin for activate cors domain in your application",
```

Please, remove `It's a pull request from ozee31/cakephp-cors that solve the issue about OPTIONS method.`
```suggestion
    "description": "A CakePHP 4 plugin for activate cors domain in your application",
```